googleworkspace/table_googleworkspace_people_directory_people.go

```diff
@@ -34,7 +34,17 @@ func listPeopleDirecoryPeople(ctx context.Context, d *plugin.QueryData, _ *plugi
 	// Define fields the API should return
 	personFields := "addresses,biographies,birthdays,calendarUrls,clientData,coverPhotos,emailAddresses,events,externalIds,genders,interests,locations,memberships,metadata,miscKeywords,names,nicknames,occupations,organizations,phoneNumbers,photos,relations,sipAddresses,skills,urls,userDefined"
 
-	resp := service.People.ListDirectoryPeople().ReadMask(personFields).Sources("DIRECTORY_SOURCE_TYPE_DOMAIN_PROFILE")
+	// By default, API can return maximum 1000 records in a single page
```

Comment:
What effect does using a larger page size vs smaller page size have on the SDK call, the Google Admin API, or anything else, i.e., response time, throttling, optimizations with context cancelling?

Does Google have any best practices on what paging size to use based on use case?

Reply:
After talking with @Subhajit97 , we've decided on going with the larger page sizes, as we'd like to reduce the number of requests when possible due to throttling and hourly/daily limits on API requests.